firstvoices/backend/management/commands/import_csv_data.py

```diff
@@ -160,7 +159,7 @@ def run_import():
 
             # raise errors to halt the import if an issue occurs
             try:
-                result = resource.import_data(dataset=table, raise_errors=True)
+                result = resource.import_data(dataset=table, raise_errors=False)
                 logger.info(
```

Comment:
Is the idea that we will only export the embedded video data? Is there anything we need to change in the import script to avoid overwriting or duplicating existing data for other models? (I'm sure you have thought about this; I'm just out of the loop.)

Reply:
The way I set it up is:
* The exporter produces CSVs for words/phrases/songs/stories, but only has rows for content that had an embedded video on it.
* The CSV only contains the ID field, the Site ID field, and the related video links field, so only those fields will be changed.
* The importer will match content on its ID. Only the site-id and related-video-links will be updated (I think it is a safe assumption that the site IDs will be the same). And the `last_modified` date will be updated to now.
* If the ID can't be matched to any content, e.g. because it's since been deleted, the importer would first try to create it, but would fail because it doesn't have enough fields, like a title. With the change in this line the script will just pass over those rows instead of crashing.

Reply:
I also tested this on the local copy of the migration I had, and didn't run into any overwriting issues. Others are welcome to test, and we should check on preprod. (unfortunately none of the sites with embedded videos were populated on dev so it will be hard to test there.)